pkgs/by-name/me/mescc-tools/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchgit
+, m2libc
+, which
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mescc-tools";
+  version = "unstable-2023-05-23";
+
+  src = fetchgit {
```

Comment:
You can use the savannah fetcher, it'll save you from downloading the git history:

```diff
diff --git a/pkgs/by-name/me/mescc-tools/package.nix b/pkgs/by-name/me/mescc-tools/package.nix
index 2c87ec13df39..dfaf152754a9 100644
--- a/pkgs/by-name/me/mescc-tools/package.nix
+++ b/pkgs/by-name/me/mescc-tools/package.nix
@@ -1,6 +1,6 @@
 { lib
 , stdenv
-, fetchgit
+, fetchFromSavannah
 , m2libc
 , which
 }:
@@ -9,11 +9,10 @@ stdenv.mkDerivation (finalAttrs: {
   pname = "mescc-tools";
   version = "unstable-2023-05-23";
 
-  src = fetchgit {
-    url = "https://git.savannah.gnu.org/git/mescc-tools.git";
+  src = fetchFromSavannah {
+    repo = "mescc-tools";
     rev = "632357f807b7a5e518d1df368d465aebc7b4d139";
     hash = "sha256-jFDrmzsjKEQKOKlsch1ceWtzUhoJAJVyHjXGVhjE9/U=";
-    fetchSubmodules = false;
   };
 
   # Don't use vendored M2libc
```